plan/refiner.go

```diff
@@ -237,6 +237,9 @@ func detachIndexScanConditions(conditions []expression.Expression, indexScan *Ph
 			filterConds = append(filterConds, cond)
 		}
 	}
+	for i, cond := range accessConds {
+		accessConds[i] = pushDownNot(cond, false)
```

Comment:
Why only do this in access condition? Because only access condition ignores NOT operator?